src/js/jquery.bxslider.js

```diff
@@ -30,6 +30,7 @@
     responsive: true,
     slideZIndex: 50,
     wrapperClass: 'bx-wrapper',
+    sliderSlopSize: 5,
```

Comment:
As I see, all current configuration options have some description in respective README [section](https://github.com/stevenwanderski/bxslider-4/blob/master/dist/README.md#configuration-options).

Could you add a short note on what the "slop" value controls (move threshold to discern between slide/click) and what dimensions (pixels) it's measured in?

Reply:
Sorry. I missed it. I didn't know. I added one, Does that sound good?

Reply:
Looks great! Hoping that any of project's maintainers can now race ahead with PR merge and new release 🤞